### PR TITLE
Configure sis-style-checksum pipeline

### PR DESCRIPTION
**Pull request by Aratz**
This PR adds a new pipeline to our sequencing-report service. This will not affect the other pipelines and can be validated at a later time, when snpseq_packs is updated.

### CODE REVIEW

File: roles/arteria-sequencing-report-ws/tasks/install.yml [hunk 1]
```diff
@@ -18,6 +18,17 @@
...
+- name: get checksum pipeline from git
+  git:
+    repo: "{{ sis_style_checksums_repo }}"
+    dest: "{{ arteria_service_scr_path }}"
+    version: "{{ sis_style_checksums_version }}"
+
+- name: copy include file
+  copy:
+    src: hiseq.rsync
+    dest: "{{ sis_style_checksums_repo }}"
```
Comment (matrulda) [inline roles/arteria-sequencing-report-ws/tasks/install.yml:30]: Shouldn't this be a location on Miarka? Maybe `arteria_service_config_root`?

File: roles/arteria-sequencing-report-ws/templates/pipeline_configs/checksum.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+nextflow_parameters:
+  config: "{{arteria_service_src_path}}/sis_style_checksums_nextflow/nextflow.config"
+  profile: "uppmax"
+environment:
+  NXF_TEMP: "{{nextflow_env.NXF_TEMP}}"
+  NXF_WORK: "{runfolder_path}/.nextflow"
+  NXF_ANSI_LOG: "false"
+pipeline_parameters:
+  input_folder: "{runfolder_path}"
+  include_file: "{{ sis_style_checksums_repo }}/hiseq.rsync"
```
Comment (matrulda) [inline roles/arteria-sequencing-report-ws/templates/pipeline_configs/checksum.yml.j2:19]: Same as previous comment.

**Review by matrulda — COMMENTED** [review]
Great work! See my comment.

File: roles/arteria-sequencing-report-ws/tasks/install.yml (hunk 1 above)
Comment (Aratz) [reply roles/arteria-sequencing-report-ws/tasks/install.yml:30]: True, good catch! Will fix!

**Review by matrulda — APPROVED** [review]
Looks good! :+1:

File: roles/arteria-sequencing-report-ws/tasks/install.yml [hunk 3]
```diff
@@ -18,6 +18,17 @@
     executable: "{{ arteria_service_env_root }}/bin/pip"
     state: present
 
+- name: get checksum pipeline from git
+  git:
+    repo: "{{ sis_style_checksums_repo }}"
+    dest: "{{ arteria_service_scr_path }}"
```
Comment (slohse) [inline roles/arteria-sequencing-report-ws/tasks/install.yml:24]: Typo

**Review by slohse — CHANGES_REQUESTED** [review]
Looks good, found just a typo.

**Pull request merged**